raiden_libs/test/mocks/client.py

```diff
@@ -78,9 +78,6 @@ def __init__(
         self.web3 = self.contract.web3
         self.transport = transport
         self.paths_and_fees = None
-        if self.transport is not None:
-            self.transport.start()  # type: ignore
```

Comment:
Is the call of `start`no needed any more?

Reply:
Nope, is working without it. Same with `pfs.transport`.